app/connectors/API1812/HIPPenaltyDetailsConnector.scala

```diff
@@ -44,21 +44,21 @@ class HIPPenaltyDetailsConnector @Inject()(val http: HttpClient, val appConfig:
       "Authorization" -> s"Bearer ${appConfig.hipToken}",
       "correlationid" -> correlationID,
       "Environment" -> appConfig.hipEnvironment,
-      "X-Originating-System" -> appConfig.hipServiceOriginatorIdKey,
+      "X-Originating-System" -> "MDTP",
       "X-Receipt-Date" -> receiptDate,
-      "X-Transmitting-System" -> appConfig.hipServiceOriginatorId
+      "X-Transmitting-System" -> "HIP"
     )
 
     val hc = headerCarrier.copy(authorization = None)
-    val url = penaltyDetailsUrl(regime)
+    val url = penaltyDetailsUrl()
 
     val hipQueryParams = Seq(
       "taxRegime" -> regime.regimeType,
       "idType" -> regime.idType,
       "idNumber" -> regime.id
     ) ++ queryParameters.toSeqQueryParams
 
-    logger.debug("[HIPPenaltyDetailsConnector][getPenaltyDetails] - " +
+    logger.info("[HIPPenaltyDetailsConnector][getPenaltyDetails] - " +
       s"Calling GET $url \nHeaders: $hipHeaders\n QueryParams: $hipQueryParams")
 
     http.GET(url, hipQueryParams, hipHeaders)(HIPPenaltyDetailsReads, hc, ec).map {
```

Comment:
NOT_FOUND is no longer relevant as a distinction, and I don't think we need this mapping since the parser is logging the result. Recover block is still relevant though